misc/xml-to-record-converter/src/main/java/io/ballerina/xmltorecordconverter/XMLToRecordRequest.java

```diff
@@ -25,53 +25,39 @@
  */
 public class XMLToRecordRequest {
 
-    private String xmlValue;
-    private boolean isRecordTypeDesc;
-    private boolean isClosed;
-    private boolean forceFormatRecordFields;
-    private String textFieldName;
-    private boolean withNameSpace;
+    private final String xmlValue;
+    private final boolean isRecordTypeDesc;
+    private final boolean isClosed;
+    private final boolean forceFormatRecordFields;
+    private final String textFieldName;
+    private final boolean withNameSpace;
 
     public XMLToRecordRequest(String xmlValue, boolean isRecordTypeDesc, boolean isClosed,
```

Comment:
Is this exposed to the tooling side? if so, modifying the constructor may cause error?

Reply:
They are sending jsonrpc request from the vscode plugin. So it won't be a problem.

Reply:
Tooling team will change the request accordingly.